packages/core-services/src/LocalBroker.ts

```diff
@@ -56,10 +69,17 @@ export class LocalBroker implements IBroker {
 		await instance.stopped();
 	}
 
-	createService(instance: IServiceClass): void {
-		const namespace = instance.getName();
+	/**
+	 * Creates a service and adds it to the local broker. In case of the broker is already started, it will start the service automatically.
+	 */
+	createService(instance: IServiceClass, serviceDependencies: string[] = []): void {
+		const namespace = instance.getName() ?? '';
```

Comment:
Would this empty string be a problem? I mean, if an instance has no name, it seems an error, isn't it?